pkg/abi/abi.go

```diff
@@ -386,17 +387,6 @@ func (e *Entry) IDBytes() []byte {
 	return id
 }
 
-// ID is a convenience function to get the ID as a hex string (no 0x prefix), which will
-// return the empty string on failure
-func (e *Entry) ID() string {
```

Comment:
Removed this, as you can just do `String()` on the `ethtypes.HexBytes0xPrefix` from `FunctionSelectorBytes()` - so this makes it consistent with `SingnatureHash()` / `SignatureHashBytes()` pattern too.